tests/openvino/tools/calibrate.py

```diff
@@ -1037,6 +1047,8 @@ def main():
     xml_path, bin_path = get_model_paths(config.model)
     accuracy_checker_config = get_accuracy_checker_config(config.engine)
     nncf_algorithms_config = get_nncf_algorithms_config(config.compression, args.output_dir)
+    if args.batch_size > 1:
+        update_config(accuracy_checker_config, args.batch_size)
```

Comment:
Why can't we update it with any `batch_size` value including 1?

Reply:
done